src/log/Log.cc

```diff
@@ -354,9 +354,12 @@ void Log::_flush(EntryQueue *t, EntryQueue *requeue, bool crash)
 void Log::_log_message(const char *s, bool crash)
 {
   if (m_fd >= 0) {
-    int r = safe_write(m_fd, s, strlen(s));
-    if (r >= 0)
-      r = safe_write(m_fd, "\n", 1);
+    size_t len = strlen(s);
```

Comment:
not sure about VLA support in c++11, is it part of the standard? or as a gnu extension?

i suggested using `alloca()` for stack allocation, which would accomplish the same thing